packages/itwin/measure-tools/src/measurements/DistanceMeasurement.ts

```diff
@@ -545,35 +545,41 @@ export class DistanceMeasurement extends Measurement {
         name: "DistanceMeasurement_Slope",
         value: fSlope,
       },
-      {
-        label: MeasureTools.localization.getLocalizedString(
-          "MeasureTools:tools.MeasureDistance.delta_x"
-        ),
-        name: "DistanceMeasurement_Dx",
-        value: fDeltaX,
-      },
-      {
-        label: MeasureTools.localization.getLocalizedString(
-          "MeasureTools:tools.MeasureDistance.delta_y"
-        ),
-        name: "DistanceMeasurement_Dy",
-        value: fDeltaY,
-      },
-      {
-        label: MeasureTools.localization.getLocalizedString(
-          "MeasureTools:tools.MeasureDistance.startCoordinates"
-        ),
-        name: "DistanceMeasurement_StartPoint",
-        value: fStartCoords,
-      },
-      {
-        label: MeasureTools.localization.getLocalizedString(
-          "MeasureTools:tools.MeasureDistance.endCoordinates"
-        ),
-        name: "DistanceMeasurement_EndPoint",
-        value: fEndCoords,
-      }
     );
+
+    if (this.drawingMetaData?.worldScale === undefined) {
```

Comment:
Wouldn't the user expect to see converted start/end coordinates when there's drawing metadata?

Reply:
Yes but, currently, we do not have the data to provide them. The conversion matrix to do so is in the works with John but, right now, we can only do scale.

Reply:
Oh, I see. I assume then we have this in our backlog?